modules/distribution/src/main/resources/log4j.properties

```diff
@@ -92,7 +92,7 @@ log4j.logger.trace.messages=TRACE,CARBON_TRACE_LOGFILE
 #log4j.logger.org.wso2.andes.server.trace=TRACE,CARBON_TRACE_LOGFILE
 
 # Uncomment to enable MessageTracer
-#log4j.logger.org.wso2.andes.tools.utils.MessageTracer=TRACE,CARBON_TRACE_LOGFILE
+log4j.logger.org.wso2.andes.tools.utils.MessageTracer=INFO,CARBON_TRACE_LOGFILE
 
```

Comment:
Does the UI pick this details from the log4j properties. How does this fix the issue? :)
Anyway we have the same issue for some other classes as well.